Application/EdFi.Ods.Api/Helpers/AssemblyLoaderHelper.cs

```diff
@@ -191,6 +206,37 @@ static bool HasPlugin(Assembly assembly)
             }
         }
 
+        private static bool IsExtensionInAssemblyModelType(string assemblyMetadataPath)
+        {
+            var assemblyMetadata = Load(assemblyMetadataPath);
+
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(TemplateSetConventions.Extension);
+        }
+
+        private static bool IsProfileInAssemblyModelType(string assemblyMetadataPath)
+        {
+            var assemblyMetadata = Load(assemblyMetadataPath);
+
+            return assemblyMetadata.AssemblyModelType.EqualsIgnoreCase(TemplateSetConventions.Profile);
+        }
+
+        public static AssemblyMetadata Load(string fileFullName)
+       {
+           if (!File.Exists(fileFullName))
+           {
+               throw new FileNotFoundException($"Non-existent file path provided. Expected location {fileFullName}.");
+           }
+
+           _logger.Debug($"Deserializing object type {typeof(AssemblyMetadata)} from file {fileFullName}.");

```

Comment:
Add single quotes around parameterized values in messages.

Reply:
yes ,  I updated . Thanks